### ISSUE TITLE
Can't mix ainput() with input() on Unix

### ISSUE THREAD

**Issue by plammens** [vxgmichel/aioconsole#99]
On Linux, trying to use regular blocking `input()` after a call to `ainput()` fails with `EOFError`.

Minimal example:
```python
import asyncio
from aioconsole import ainput


async def main():
    await ainput("ainput: ")
    input("input: ")


asyncio.run(main())
```

On Ubuntu 20.04, if you run this and enter something at the first prompt (`ainput: `), then the `input()` call fails immediately with `EOFError`.
On the other hand, this works fine on Windows.

If the `input()` call is made before `ainput()`, then there are no problems.

**Comment by vxgmichel** [comment]
Hi @plammens and thanks for your report!

We already had a similar discussion here: https://github.com/vxgmichel/aioconsole/issues/90#issuecomment-997997618

Still, your suggestion about setting the non-blocking mode only when necessary is interesting, I'll think about it :) 

However and as I said in the comment mentioned above, I would advise to simply avoid using `input()` from an asyncio coroutine as it would block the event loop while waiting for the user input.

**Comment by plammens** [comment]
Thanks @vxgmichel, sorry hadn't seen #90, it's the same issue!

I also wanted to add that I've found that when using `ainput`, you shouldn't use regular `input` but neither regular `print`, the latter because it risks raising a BlockingIOError if the buffer is full (e.g. if you try to print a lot of stuff) since stdout is in non-blockikg mode.

```python
await ainput()
print('a' * 10**7)  # BlockingIOError
```

So currently, if you use `ainput` once you should always use `ainput`/`aprint` and never `input`/`print`.

**Comment by plammens** [comment]
Regarding my suggestion, it might lead to problems if multiple `ainput`s are being awaited simultaneously:

1. The first `ainput` sets non-blocking mode and starts waiting for input
2. The first `ainput` sets non-blocking mode (no-op) and starts waiting for input
3. User enters input
3. The first `ainput` returns, sets blocking mode

Would that mess up the second one, which is still waiting?

**Comment by vxgmichel** [comment]
> Would that mess up the second one, which is still waiting?

Yes it would, so the implementation would have to take care of that. Another solution would be to patch `builtins.print` and `builtins.input` to add warning but patching stdlib is often more annoying than helpful. I'm not sure what to do about this issue :thinking:
